src/ui/components/shared/CommentTool.tsx

```diff
@@ -110,7 +110,7 @@ function CommentTool({
     }
 
     const pendingCommentEditorId = getCommentEditorDOMId(pendingComment.comment);
-    const isEditorFocused = !!document.activeElement.closest(`#${pendingCommentEditorId}`);
+    const isEditorFocused = !!document.activeElement.closest(`[id="${pendingCommentEditorId}"]`);
```

Comment:
yikes.

Reply:
@jasonLaster turns out our comment ids can have "="s in them, e.g. `Yzo1ZDk2NzAzNy03Y2EzLTRiODQtYWYyMi05MWJlNGIwNzM1YTc=` and if you try to do...

```js
document.closest(`#comment-editor-Yzo1ZDk2NzAzNy03Y2EzLTRiODQtYWYyMi05MWJlNGIwNzM1YTc=`)
```

... you get

```exception
VM1480:1 Uncaught DOMException: Failed to execute 'closest' on 'Element': '#comment-editor-Yzo1ZDk2NzAzNy03Y2EzLTRiODQtYWYyMi05MWJlNGIwNzM1YTc=' is not a valid selector.
```

I switched to `[id="..."]` to quote out that equals sign.

Reply:
9a5cffc